CIP-????/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: 89
```

Reply:
Committing so we can become more ambitious about cross-referencing this with other CIPs and CPS-0004? https://github.com/cardano-foundation/CIPs/pull/418